src/vm/eventpipe.h

```diff
@@ -27,6 +27,14 @@ struct EventData;
 
 typedef uint64_t EventPipeSessionID;
 
+enum class EventPipeState : uint32_t
+{
+    NotInitialized,
+    FailedToInitialize,
```

Comment:
Is it important to distinguish 'FailedToInitialize' differently than 'NotInitialized'? We never made this distinction in the past and I wasn't aware of any issues caused by that.

Reply:
The distinction is to check that Initialized was called once, but not necessary. I can remove it.

Reply:
I would : )